modules/installing-rhv-accessing-the-ocp-web-console.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_rhv/installing-rhv-default.adoc
+// * installing/installing_rhv/installing-rhv-custom.adoc
+
+[id="installing-rhv-accessing-the-ocp-web-console_{context}"]
+= Accessing the the {product-title} web console
+When the {product-title} is up and running, log into the {product-title} web console.
+
+.Procedure
+. Optional: In the {rh-virtualization-first} Administration Portal, open *Compute* -> *Cluster*.
+. Verify that the installation program creates the virtual machines for the {product-title} nodes.
```

Comment:
> s/creates/created
> s/nodes/machines
> What does this step have to do with accessing the OCP web console?

- "Creates" (present) because the customer is likely to be observing the Admin Portal while the installation program creates the virtual machines. 
- "Machines"  because the RHV Admin Portal uses the term, "virtual machine." 
- Before going to the ocp web console, the customer verifies that the virtual machines for the cluster/control plane are running. Depending on the RHV environment, this might take a while.